js/mediaView.js

```diff
@@ -0,0 +1,604 @@
+          '.mejs-captions-button button' :
+          '.mejs-captions-selector';
+        this.$(selector).off('click.mediaCaptionsChange');
+      }
+
+      const modelOptions = this.model.get('_playerOptions');
+      delete modelOptions.success;
+
+      const media = this.model.get('_media');
+      if (media) {
+        switch (media.type) {
+          case 'video/vimeo':
+            this.$('iframe')[0].isRemoved = true;
+        }
+      }
+
+      if (this.mediaElement && this.mediaElement.player) {
+        const playerId = this.mediaElement.player.id;
+
+        purge(this.$el[0]);
+        this.mediaElement.player.remove();
+
+        if (window.mejs.players[playerId]) {
+          delete window.mejs.players[playerId];
+        }
+      }
+
+      if (this.mediaElement) {
+        $(this.mediaElement).off({
+          'play': this.onMediaElementPlay,
+          'pause': this.onMediaElementPause,
+          'ended': this.onMediaElementEnded,
+          'seeking': this.onMediaElementSeeking,
+          'timeupdate': this.onMediaElementTimeUpdate,
+          'inview': this.onMediaElementInview
+        });
+
+        this.mediaElement.src = '';
+        $(this.mediaElement.pluginElement).remove();
+        delete this.mediaElement;
+      }
+
+      super.remove();
+    }
+
+    onDeviceChanged() {
+      if (this.model.get('_media').source) {
+        this.$('.mejs-container').width(this.$('.component__widget').width());
+      }
+    }
+
+    onPlayerReady(mediaElement, domObject) {
+      this.mediaElement = mediaElement;
+
+      const player = this.mediaElement.player;
+      if (!player) player = window.mejs.players[this.$('.mejs-container').attr('id')];
+
+      const hasTouch = window.mejs.MediaFeatures.hasTouch;
+      if (hasTouch) {
+        this.setupPlayPauseToggle();
+      }
+
+      this.addThirdPartyAfterFixes();
+      this.cleanUpPlayerAfter();
+
+      if (player && this.model.has('_startVolume')) {
+        // Setting the start volume only works with the Flash-based player if you do it here rather than in setupPlayer
+        player.setVolume(parseInt(this.model.get('_startVolume')) / 100);
+      }
+
+      this.setReadyStatus();
+      this.setupEventListeners();
+    }
+
+    addThirdPartyAfterFixes() {
+      const media = this.model.get('_media');
+      switch (media.type) {
+        case 'video/vimeo':
+          this.$('.mejs-container').attr('tabindex', 0);
+      }
+    }
+
+    cleanUpPlayerAfter() {
+      this.$("[aria-valuemax='NaN']").attr('aria-valuemax', 0);
+    }
+
+    onScreenSizeChanged() {
+      this.$('audio, video').width(this.$('.component__widget').width());
+    }
+
+    onSkipToTranscript() {
+      // need slight delay before focussing button to make it work when JAWS is running
+      // see https://github.com/adaptlearning/adapt_framework/issues/2427
+      _.delay(() => {
+        this.$('.media__transcript-btn').a11y_focus();
```

Comment:
Change to new a11y API, I think it should be:
```suggestion
        Adapt.a11y.focus(this.$('.media__transcript-btn'));
```